challengeFinalComponents/HUDView/HUDView.swift

```diff
@@ -8,22 +8,22 @@
 import Foundation
 import UIKit
 
-class HUDView: UIStackView{
+class HUDView: UIView {
     
     private lazy var pauseButton: PauseButton = {
         let pause = PauseButton()
         pause.translatesAutoresizingMaskIntoConstraints = false
+        self.addSubview(pause)
         return pause
     }()
     
-    private lazy var decisionPointsView: DecisionPointsView = {
-        let decision = DecisionPointsView()
-        decision.translatesAutoresizingMaskIntoConstraints = false
+    private lazy var decisionPointsView: PointsView = {
+        let decision = PointsView()
+        self.addSubview(decision)
```

Comment:
isn't it missing decision.translatesAutoresizingMaskIntoConstraints = false  ?

Reply:
ta dentro da pointsView